src/coaches/coaches.controller.ts

```
@@ -0,0 +1,23 @@
+
+@Controller('coaches')
+export class CoachesController {
+    constructor(private readonly coachesService: CoachesService) {}
+    
+    // @UseInterceptors(LoggingInterceptor)
+    @Get()
+    getAll(){
+        return this.coachesService.findAll();
+    }
+    @Get('id/:id')
+    getById(@Param('id', ParseUUIDPipe) id: UUID) { 
```

ESLint

[error] Replace `··getById(@Param('id',·ParseUUIDPipe)·id:·UUID)·{·` with `getById(@Param('id',·ParseUUIDPipe)·id:·UUID)·{`

(prettier/prettier)